storage/interceptors/smaap_service_binding_interceptor.go

```diff
@@ -166,20 +166,22 @@ func (i *ServiceBindingInterceptor) AroundTxCreate(f storage.InterceptCreateArou
 					StatusCode:  http.StatusBadGateway,
 				}
 				if shouldStartOrphanMitigation(err) {
-					// store the instance so that later on we can do orphan mitigation
-					_, err := f(ctx, obj)
-					if err != nil {
-						return nil, fmt.Errorf("broker error %s caused orphan mitigation which required storing the resource which failed with: %s", brokerError, err)
-					}
-
-					// mark the operation as deletion scheduled meaning orphan mitigation is required
```

Comment:
store service instanceID always you are trying to fetch the binding from repository anyway

Reply:
hmm it seem to be out dated?
https://github.com/Peripli/service-manager/blob/b1f4fc5b925343a3fcf3bd60e0f8d938130eb1a8/storage/interceptors/smaap_service_binding_interceptor.go#L179